sdcm/utils/common.py

```diff
@@ -1816,3 +1817,9 @@ def get_docker_stress_image_name(tool_name=None):
         result = image_file.read()
 
     return result.strip()
+
+
+class ScyllaConfigParameters(enum.Enum):
```

Comment:
What is the benefit of using the enum here?

Reply:
Keep the list of  Scylla configuration parameters in the one place in the code, don't use hardcoded names.
I used enum because of it's used for unique sets definition

Reply:
I think it doesn't help much,

1) using it looks like that and it rename the parameter, plus you need to keep in mind all the time you need to call value..
ScyllaConfigParameters.REPAIR_BASED_OPS.value
2) scylla configuration is documented in one place, making a mirror image of some of it in our code might counter productive at some point

3) those configuration rarely change, and are consider, mostly adding more and more of it

Reply:
I agree with Israel on this, starting duplicating the names of Scylla parameters here don’t add any added value IMO

Reply:
removed